CHANGELOG.rst

```diff
@@ -2,6 +2,19 @@
 Changelog
 =========
 
+Version 5.0.2
+-------------
+This release was created to auto-publish the binaries for watem-sedem on GitHub.
+
+For windows: watem_sedem.exe
+For linux: watem_sedem
+For debian/ubuntu linux als on apt package is provided.
+
+Version 5.0.1
+-------------
+This release contains some small updates to the documentation and all
+extensions are disabled by default.
```

Comment:
extension are disabled by default: I suppose in the code, not in the predefined ini-files?